### PR TITLE
Add button to create a meta-issue on GitHub

### PR DESCRIPTION
**Pull request by tidoust**
This adds a button to the "How to get horizontal review" section to generate a meta-issue to track horizontal review steps in a GitHub repository, following a suggestion made by @anssiko in:
https://github.com/w3c/openscreenprotocol/issues/284#issuecomment-849758776

Some notes:
- Taking that approach instead of creating an issue template, as I believe that the issue template would be seen whenever someone tries to create a new issue and the horizontal reviews issue will only be useful once or twice per specification.
- Sub-details sections are not included because they would grow the length of the body beyond what can be passed to GitHub in a query string
- To keep script simple, actual steps need to be made explicit by enclosing the prose for each step into a `<span class="step">` construct.

Here is a screenshot of the section that this code adds to the document:
Screenshot 2021-06-07 at 23-15-16 Document Review

And here is a screenshot of the issue that would be created:
Screenshot 2021-06-07 at 23-17-34 w3c openscreenprotocol

### CODE REVIEW

**Comment by anssiko** [comment]
@tidoust LGTM, thank you for this! 🎉

I'll test drive this live when you deploy. It makes sense to take this approach instead of an issue template. It was probably more work for you, but the nice integration into the "How to get horizontal review" is worth it.

**Comment by r12a** [comment]
I like the idea of this. There are a couple of conflicts due to recent changes which better indicate that horizontal review is a part of wide review, and this page covers both (in response to an issue that was raised).

To resolve the conflict, remove the bullet points

• Groups listed in your charter, and 
• General announcement

But that seems consistent with the idea that this is creating a 'meta-issue to track horizontal review steps'.

I especially like the way that paragraph text has been changed to itemised lists for each of the HR functions – i already had it in mind to do something similar, in fact.

@samuelweiler @plehegar do you agree to merge this after resolving the conflicts?

File: index.html [hunk 1]
```diff
@@ -146,14 +157,25 @@ <h2><a href="#how_to_get_horizontal_review">How to get horizontal review</a></h2
...
+<dd><span class="step">Use <a rel="nofollow" class="external text" href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a> for general announcements regarding wide and horizontal reviews. (Sending an email to this list alone is not sufficient to trigger horizontal reviews. You will still need to formally request review from the Horizontal Groups, as described above.)</span></dd>
 </dl>
 
+<details id="githubissue" hidden>
+  <summary>Generate a meta-issue to track horizontal review steps in a GitHub repository</summary>
+  <p>You may find it useful to create an issue in the GitHub repository that hosts a specification that needs to get horizontal reviews to track the different steps involved to request these reviews. Fill the name of your GitHub repository in the field below and hit the "Create GitHub issue" button to open the "new issue" form on the specified GitHub repository, with the issue's body pre-filled with horizontal review steps as a list of checkboxes.</p>
```
Comment (r12a) [inline index.html]: This is a little difficult to parse.  Here's a suggestion for replacement text:

You may find it useful to create an issue in the GitHub repository of your spec to track your progress. Add the name of your GitHub repository to the field below and hit the "Create GitHub issue" button. This opens the "new issue" form in your repository, and pre-fills the body with horizontal review steps as a list of checkboxes.

**Comment by tidoust** [comment]
Sorry, that took some time, I had left that on the side and, ahem, forgot about it. Anyway, I resolved the conflicts and integrated your suggestion, @r12a.

**Comment by plehegar** [comment]
LGTM

**Pull request merged**